R/init.R

```diff
@@ -37,7 +37,11 @@ init_site <- function(pkg = ".") {
   rule("Initialising site")
   dir_create(pkg$dst_path)
   copy_assets(pkg)
-  copy_favicons(pkg)
+
+  if (has_logo(pkg)) {
```

Comment:
I feel like this should be `if (has_logo(pkg) && !has_favicons(pkg))`, and then you can simplify the logic in `build_favicons()`